CHANGELOG.md

```diff
@@ -92,7 +92,6 @@
 
 ## [unreleased]
 
-- Activity to Activity association renamed `child_activities` (from `activities`) to avoid an association name clash with the `public_activity` gem
```

Comment:
Oop! I think I've already seen this back in 815c816cb3cfa21d013769d9a6ad2c92389ebc51

Reply:
Oops, merge mistake!